memeolist/app/src/main/java/org/aerogear/android/app/memeolist/ui/CommentsFormActivity.java

```diff
@@ -79,26 +75,30 @@ void newComment(View view) {
                 memeid(comment.getMemeId())
                 .build();
 
-        apolloClient.mutate(postCommentMutation)
-                .enqueue(new ApolloCall.Callback<PostCommentMutation.Data>() {
+        SyncClient
+                .getInstance()
+                .mutation(postCommentMutation)
+                .execute(PostCommentMutation.Data.class)
+                .respondOn(new AppExecutors().mainThread())
```

Comment:
I know that this is SDK thing but can this be optional and main thread will be default.

Reply:
I will think in a way to do that. Because the `execute` is `.respondOn(new AppExecutors().networkThread())` so we need to switch the `respondWith` to the mainThread

Reply:
Well if you don't specify it it is undefined.

If the library doesn't specify a request thread it will run on the calling thread (in this case main).  In the current implementation if a thread is defined for requestOn but not respondOn then it will run on the request thread.  This is the same behavior in RxJava.